ts/features/payments/checkout/store/actions/networking.ts

```diff
@@ -43,7 +43,7 @@ export const paymentsCalculatePaymentFeesAction = createAsyncAction(
   "PAYMENTS_CALCULATE_PAYMENT_FEES_SUCCESS",
   "PAYMENTS_CALCULATE_PAYMENT_FEES_FAILURE"
 )<
-  CalculateFeeRequest & { paymentMethodId: string },
+  CalculateFeeRequest & { paymentMethodId: string; idPsp?: string },
```

Comment:
This payload is growing, we should create a dedicated type. What do you think?

Reply:
Addressed into 83ea326eff53a9f4075966e9551a0d35c1ca18e3